packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -42,13 +44,28 @@ export interface IDropdownProps {
   onChanged?: (option: IDropdownOption, index?: number) => void;
 
   /**
-   * Optional custom renderer for the selected dropdown item
+   * Optional custom renderer for selected option displayed in input
    */
-  onRenderItem?: IRenderFunction<IDropdownOption>;

```

Comment:
oh yeah...that was an original renderable. David and I changed it for clarity. Forgot it was already in the props. I'll revert.